### ISSUE TITLE
Big space on the left and right side of the editor panels

### ISSUE THREAD

**Issue by octohedron** [Microsoft/vscode#53778]
Issue Type: <b>Bug</b>

This wasn't happening on the previous version, pretty sure it's something from the new release.

Steps to reproduce:
+ Get a long monitor, this one is 21:9 
+ Install vscode
+ Split pane horizontally

Now you have big, unused, unrequested spaces on the left and right side of your editor window, as shown in the image

![unwanted spacing in vscode left and right](https://i.imgur.com/iGLCpYv.jpg)

VS Code version: Code 1.25.0 (0f080e5267e829de46638128001aeb7ca2d6d50e, 2018-07-05T13:02:28.333Z)
OS version: Darwin x64 17.6.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz (8 x 2900)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>vpx_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (4.45GB free)|
|Process Argv|/Applications/Visual Studio Code.app/Contents/MacOS/Electron|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (11)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|1.4.12
vsc-material-theme|Equ|2.3.0
prettier-vscode|esb|1.5.0
vscode-pyvmmonitor|fab|0.0.3
gc-excelviewer|Gra|2.1.24
contextualduplicate|laf|0.2.0
prettify-json|moh|0.0.3
python|ms-|2018.6.0
Go|ms-|0.6.84
vscode-icons|rob|7.24.0
vscode-react-native|vsm|0.6.11

(2 theme extensions excluded)

</details>

**Comment by octohedron** [comment]
Seems to have fixed itself, so closing it for now, will re-open if it happens again.

**Comment by octohedron** [comment]
It just came back after opening a new window.

It was OK until I opened a second editor (Shift + Command + N) and opened a new folder.

Tried closing it, restarting, etc, still the same.

It's also possible to resize the margin but it will never go to less than the space shown in the screenshot.

![screen shot 2018-07-13 at 12 28 07](https://user-images.githubusercontent.com/7332540/42687215-7daeadf4-8698-11e8-877e-f347f26c5812.png)
